src/main/java/org/aksw/word2vecrestful/subset/DataSubsetProvider.java

```diff
@@ -0,0 +1,84 @@
+package org.aksw.word2vecrestful.subset;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.aksw.word2vecrestful.utils.Cfg;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+/**
+ * Class to help retrieve the list of words from a subset stored on the disk
+ * 
+ * @author Nikit
+ *
+ */
+public class DataSubsetProvider {
+
+	public static String fileDir = Cfg.get("org.aksw.word2vecrestful.Application.subsetfiledir");
+	public static final Map<String, List<String>> SUBSET_MODELS = new HashMap<>();

```

Comment:
I am slightly unhappy with this. Either this class itself is a singleton OR you have an instance that you are distributing via dependency injection. However, now you have made everything static which is not a good way to program it.